diff -Naur dvdisaster-0.79.3/tools/pngpack.c dvdisaster-0.79.3.new/tools/pngpack.c
--- dvdisaster-0.79.3/tools/pngpack.c   2010-11-11 11:38:56.000000000 +0100
+++ dvdisaster-0.79.3.new/tools/pngpack.c       2015-04-12 16:07:00.502601593 +0200
@@ -29,6 +29,10 @@
 #include "decimate.h"
 #include "memory.h"
 
+#if defined(PNG_LIBPNG_VER_MINOR) && PNG_LIBPNG_VER_MINOR >= 5
+# include <zlib.h>
+#endif
+
 /***
  *** main()
  ***/
